cmd/podman/kube/play.go

```diff
@@ -364,31 +364,53 @@ func playKube(cmd *cobra.Command, args []string) error {
 	return play(cmd, args)
 }
 
-func readerFromArg(fileName string) (*bytes.Reader, error) {
-	var reader io.Reader
-	switch {
-	case fileName == "-": // Read from stdin
-		reader = os.Stdin
-	case parse.ValidWebURL(fileName) == nil:
-		response, err := http.Get(fileName)
+func readerFromArgs(args []string) (*bytes.Reader, error) {
+	return readerFromArgsWithStdin(args, os.Stdin)
+}
+
+func readerFromArgsWithStdin(args []string, stdin io.Reader) (*bytes.Reader, error) {
+	// if user tried to pipe, shortcut the reading
+	if len(args) == 1 && args[0] == "-" {
+		data, err := io.ReadAll(stdin)
 		if err != nil {
 			return nil, err
 		}
-		defer response.Body.Close()
-		reader = response.Body
-	default:
-		f, err := os.Open(fileName)
+		return bytes.NewReader(data), nil
+	}
+
+	var combined bytes.Buffer
+
+	for i, arg := range args {
+		reader, err := readerFromArg(arg)
+		if err != nil {
+			return nil, err
+		}
+
+		_, err = io.Copy(&combined, reader)
+		reader.Close()
 		if err != nil {
 			return nil, err
 		}
-		defer f.Close()
-		reader = f
+
+		if i < len(args)-1 {
+			combined.WriteString("\n---\n")
```

Comment:
Please add a comment on top that this is file seperator ? Maybe also have `\n---\n` in a const string